### ISSUE TITLE
How to Utils.matToTexture2D for Float types?

### ISSUE THREAD

**Issue by ivilkhovyiameria** [EnoxSoftware/OpenCVForUnity#192]
I need to convert the result of calcOpticalFlowFarneback CV_32FC2 to Unity texture, e. g. TextureFormat.RGFloat, but when I use Utils.matToTexture2D I see the error "The Mat object must have the types 'CV_8UC4' (RGBA) , 'CV_8UC3' (RGB) or 'CV_8UC1' (GRAY)." So it says that I am forced to only three formats, and only uint. What can I do to work this around? Later in the code I am passing the data into compute shader as a texture. Thank you.

**Comment by ivilkhovyiameria** [comment]
This is exactly what I needed. Thank you.
I believe the solution was not very clear from the documentation or docstrings. I was about to give up before I wrote here. When I saw fastMatToTexture2D I would never thought that it supports broader formats than "non-fast" version. Or when I saw the error message "The Mat object must have the types 'CV_8UC4' (RGBA) , 'CV_8UC3' (RGB) or 'CV_8UC1' (GRAY).", it was not clear that I can use another method.
